en/editors_and_tools/creator_to_cocos2dx.md

```diff
@@ -3,3 +3,18 @@
 </div>
 
 ## Using Cocos Creator With C++ and Lua Projects
+__Cocos Creator__ supports JavaScript, built in. Edit your `Scenes` and source code all from within. However, If you are a C++ or Lua developer, __Creator__ allows exporting of `Scenes` to sour code for further development. Why isn't C++ built in, you ask? There is no need to re-invent the wheel. There are many really good development environments out there. Writing a text editor is no trivial task. For this reason, it is best to allow developers to edit source code in the editor of their choice.
+
+### What Is Supported?
```

Comment:
Take a look at these headings. Am I covering everything?

Reply:
I think these things can be added too:

- how to run the demo project that i have done
- how to integrate exported source code and resources in new projects
- how to use AnimationManager
- how to use ColliderManager
- what cocos2d-x versions and Creator versions supported

So may be there should be more docs for it.

Reply:
I mean, this doc is good for basic introduction, but may need more docs about it.

Reply:
@minggo Yup, I am adding all of that.

Reply:
Awesome. Please let me know if you need any help.
Thanks.